internal/http/tmdb/search.go

```diff
@@ -299,24 +299,24 @@ type (
 		StatusMessage string `json:"status_message"`
 	}
 	FailedRequestError struct {
-		httpCode int
-		tmdbCode int
-		message  string
+		HttpCode int
+		TmdbCode int
+		Message  string
```

Comment:
Why are these public now?

Reply:
Hm, that's a good question. I've made them private (and in the other errors too). Hopefully nothing subtle breaks, but from what I can tell the `String()` implementation is all that uses the error fields (which is internal to the package anyway).